pkg/controller/provider/container/vsphere/reconciler.go

```diff
@@ -860,11 +860,6 @@ func (r Reconciler) applyEnter(tx *libmodel.Tx, u types.ObjectUpdate) error {
 		return liberr.Wrap(err)
 	}
 
-	r.log.V(3).Info(
-		"Model created.",
-		"model",
-		libmodel.Describe(m))
-
```

Comment:
Level=3 should not be used for per-object logging.
This will be logged by the _model_ layer at level=4.